openstack/cinder/templates/type-seeds.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Release.IsUpgrade }}
```

Comment:
I don't understand, why we need this. Does this mean, we don't want to put this into kubernetes on first deployment? If yes, why?

Reply:
removed